src/ui/tutorial/tutorial.gss

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2017 The Coding with Chrome Authors.
+ * Copyright 2018 The Coding with Chrome Authors.
```

Comment:
The year is normally only set once when the file is created and doesn't need to be updated later.

Reply:
This one was originally created in 2018. 2017 just from copying another file, so this is a correction.